### PR TITLE
hw-2-lottery

### PR DESCRIPTION
**Pull request by VolodymyrDavidov**

### CODE REVIEW

File: src/main/java/core/basesyntax/Color.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,
```
Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/Color.java:4]: Enum members are constants. You should use uppercase here

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```
Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]: It's better to use `name()` for getting String representation of enum constants

Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]: `Color.values()` can be extracted as separate variable, as it's used twice

**Review by uliana-andrushchuk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] getColor = Color.values();
+        int index = new Random().nextInt(getColor.length);
+        return getColor[index].name();
```
Comment (aleksandr-bevza) [inline src/main/java/core/basesyntax/ColorSupplier.java:9]: 1. getColor is a good name for method, but not for variable.
2. Move Random to class field. Don't create new Random each time this method is invoked.
```suggestion
        Color[] colorValues = Color.values();
        int index = random.nextInt(colorValues.length);
        return getColor[index].name();
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER = 100;
```
Comment (aleksandr-bevza) [inline src/main/java/core/basesyntax/Lottery.java:6]: Variable's name should explain it's purpose
```suggestion
    private static final int NUMBER_LIMIT = 100;
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
```
Comment (aleksandr-bevza) [inline src/main/java/core/basesyntax/Lottery.java:9]:
```suggestion
```

**Review by aleksandr-bevza — CHANGES_REQUESTED** [review]

**Pull request closed without merging**